src/main/Core/SingleInstanceLock/SingleInstanceLockModule.ts

```diff
@@ -1,10 +1,13 @@
-import type { App } from "electron";
+import type { Dependencies } from "@Core/Dependencies";
+import type { DependencyRegistry } from "@Core/DependencyRegistry";
 
 export class SingleInstanceLockModule {
-    public static bootstrap(app: App) {
-        if (!app.requestSingleInstanceLock()) {
-            console.log("Quitting application. Reason: another instance is already running");
-            app.quit();
-        }
+    public static bootstrap(dependencyRegistry: DependencyRegistry<Dependencies>) {
+        const eventEmitter = dependencyRegistry.get("EventEmitter");
+        const app = dependencyRegistry.get("App");
+
+        app.on("second-instance", () => {
+            eventEmitter.emitEvent("hotkeyPressed");
+        });
```

Comment:
I get what you're trying to achieve here but this is a very bad solution to the problem.

As the name of the event "hotkeyPressed" is pretty clear, **it should only be emitted when the hotkey is actually pressed**. Because that's not the case here this event should not be emitted.

Reply:
Ah sorry don't know how I managed to overlook that. I'll clean up the code